server/src/test/scala/sbt/server/ProtocolTest.scala

```diff
@@ -761,137 +446,6 @@ class ProtocolTest {
-
-    // compilation
-    //oneWayTrip(compilation) { _ / "compilation" / "compilation.json" }
-    //oneWayTrip(protocol.Compilations(Vector(compilation))) { _ / "compilation" / "compilations.json" }
-
-    // access
-    //oneWayTrip(protocol.Public: protocol.Access) { _ / "access" / "public.json" }
-    //oneWayTrip(protocol.Protected(protocol.Unqualified): protocol.Access) { _ / "access" / "protected.json" }
-    //oneWayTrip(protocol.Private(protocol.Unqualified): protocol.Access) { _ / "access" / "private.json" }
-    //oneWayTrip(protocol.Protected(protocol.ThisQualifier): protocol.Access) { _ / "access" / "protected_this.json" }
-    //oneWayTrip(protocol.Private(protocol.ThisQualifier): protocol.Access) { _ / "access" / "private_this.json" }
-    //oneWayTrip(protocol.Protected(protocol.IdQualifier("foo")): protocol.Access) { _ / "access" / "protected_foo.json" }
-    //oneWayTrip(protocol.Private(protocol.IdQualifier("foo")): protocol.Access) { _ / "access" / "private_foo.json" }
-
-    // type
-    //val foo_tpe = protocol.Singleton(protocol.Path(Vector(protocol.Id("Foo"))))
-    //oneWayTrip(tpe: protocol.Type) { _ / "type" / "singleton.json" }
-    //oneWayTrip(protocol.Singleton(protocol.Path(Vector(protocol.Super(protocol.Path(Vector(protocol.Id("Foo"))))))): protocol.Type) { _ / "type" / "super_foo.json" }
-    //oneWayTrip(protocol.Singleton(protocol.Path(Vector(protocol.This))): protocol.Type) { _ / "type" / "singleton_this.json" }
-    //oneWayTrip(protocol.Projection(protocol.Singleton(protocol.Path(Vector(protocol.This))), "Foo"): protocol.Type) { _ / "type" / "projection.json" }
-    //oneWayTrip(
-    //  protocol.Parameterized(foo_tpe,
-    //    Vector(protocol.ParameterRef("A"))): protocol.Type) { _ / "type" / "parameterized.json" }
-    //oneWayTrip(protocol.EmptyType: protocol.Type) { _ / "type" / "empty_type.json" }
-    //oneWayTrip(protocol.Annotated(tpe, Vector(annotation)): protocol.Type) { _ / "type" / "annotated.json" }
-    //oneWayTrip(protocol.Structure(Vector(tpe), Vector(definition), Vector(definition)): protocol.Type) { _ / "type" / "structure.json" }
-    //oneWayTrip(protocol.Polymorphic(foo_tpe, Vector(typeParam)): protocol.Type) { _ / "type" / "polymorphic.json" }
-    //oneWayTrip(protocol.Existential(foo_tpe, Vector(typeParam)): protocol.Type) { _ / "type" / "existential.json" }
-    //oneWayTrip(protocol.Constant(foo_tpe, "A"): protocol.Type) { _ / "type" / "constant.json" }
-
-    // byte array
-    //oneWayTrip(ByteArray(Array(0.toByte))) { _ / "byte_array" / "byte_array.json" }
-
-    // analysis
-    // FIXME which fields do we need in analysis
-    //oneWayTrip(protocol.Analysis()) { _ / "analysis" / "analysis.json" }
-    /*    oneWayTrip(protocol.Analysis(
-      stamps = stamps, apis = apis, relations = relations, infos = infos,
-      compilations = protocol.Compilations(Vector(compilation)))) { _ / "analysis" / "analysis.json" }
```

Comment:
Json files are written out using the oneway test code itself, and the current sets of json files were written after we migrated to pickling-based so they seem to be gone. Not sure if it's using the latest json writer since I'm still seeing decimal points like `12.0` in `long.json`. @jsuereth?